### DISCUSSION TITLE
GitHub Actions has encountered an internal error when running your job.

### DISCUSSION THREAD

**Discussion by leahcimto** [upptime/upptime#362] [Q&A] [answered]
This has been working but then it started saying "GitHub Actions has encountered an internal error when running your job.", and my other upptime repos are working fine. Does anyone know why this is?

**Comment by AnandChowdhary** [comment, marked as answer]
Yes, GitHub Actions was down today: https://www.githubstatus.com/incidents/zbpwygxwb3gw
